device/mellanox/x86_64-nvidia_sn4280-r0/ACS-SN4280/README.md

```diff
@@ -0,0 +1 @@
+This SKU is dedicated to Smart Switch
```

Comment:
Why is this file needed? I don't see it in other SKUs

Reply:
removed